package.json

```diff
@@ -34,7 +34,8 @@
     "dev-start": "nodemon -e js,jsx,md,mustache -w ./src --exec ./dev-tools/babel-run -- ./src/server",
     "generate-contacts": "./dev-tools/babel-run ./dev-tools/generate-contacts.js",
     "jest-test": "jest __test__/backend.test.js",
-    "dev": "nf start -w --procfile ./dev-tools/Procfile.dev"
+    "dev": "npm run tunnel | nf start -w --procfile ./dev-tools/Procfile.dev",
+    "tunnel": "node ./tunnel.js"
```

Comment:
I think we want something like `if [ "$NGROK_AUTH_TOKEN" != \"\" ] ; then node ./tunnel.js; fi` for the tunnel job -- basically it should be able to work without ngrok.